doc/installing.rst

```diff
@@ -6,6 +6,15 @@ Installation
 Required dependencies
 ---------------------
 
+.. warning::
+
+  Xarray is dropping support for Python 2 at the end of 2018. For more
+  information see the following references:
+
+  - `Xarray Github issue discussing dropping Python 2 <https://github.com/pydata/xarray/issues/1829>`__
+  - `Python 3 Statement <http://www.python3statement.org/>`__
+  - `Tips on porting to Python 3 <https://docs.python.org/3/howto/pyporting.html>`__
+
 - Python 2.7, 3.4, 3.5, or 3.6
```

Comment:
Instead of a warning, maybe this should be written as a footnote after 2.7? We can save the loud "Warning" for the release notes.

Also, can we say "plans to drop support" instead of "is dropping support"?